src/components/CategoryItem.js

```diff
@@ -0,0 +1,152 @@
+import * as React from 'react';
+import { redirect } from 'next/navigation';
+import styled from 'styled-components';
+import arrow from 'public/images/arrow.png';
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+
+export default function CategoryItem({ name, link, size }) {
+    const router = useRouter();
+    const handleClick = () => {
+        router.push(
+            router.push({
+                pathname: '/assets/' + link ,
+            }),
+        );
+    };
+
+    return (
+        <ButtonStyle onClick={handleClick}>
+            <ButtonStyleTop>
+                <ButtonStyleTextWrapper>
+                    <ButtonStyleTextSize>{size}</ButtonStyleTextSize>
+                    <ButtonStyleTextName>{name}</ButtonStyleTextName>
+                </ButtonStyleTextWrapper>
+            </ButtonStyleTop>
+            <ButtonStyleBottom>
+                <ButtonLinkStyleTextWrapper>
+                    <ButtonLinkStyleText>View All</ButtonLinkStyleText>
+                    <ArrowStyle>
+                        <img src={'/images/arrow.png'} alt="arrow.png" />
```

Comment:
이미지는 Next/Image를 사용하시고, import 하여 사용해주시길 바랍니다.

> Please use Next/Image for the image and import it.

Reply:
alt를 작성할 때는 png 같은 불필요한 정보는 빼주는 게 좋습니다. 아래 가이드를 참고해서 다시 적어보면 좋을 것 같습니다.

> When writing alt, it is better to exclude unnecessary information such as png. I think it would be good to write it down again by referring to this guide.

https://help.siteimprove.com/support/solutions/articles/80000863904